apps/mobile/src/components/Feed/CommentsBottomSheet/CommentsBottomSheetList.tsx

```diff
@@ -1,36 +1,57 @@
 import { FlashList, ListRenderItem } from '@shopify/flash-list';
-import { useCallback } from 'react';
-import { View } from 'react-native';
+import { useCallback, useEffect, useRef } from 'react';
 import { graphql, useFragment } from 'react-relay';
 
 import { CommentsBottomSheetLine } from '~/components/Feed/CommentsBottomSheet/CommentsBottomSheetLine';
 import { CommentsBottomSheetList$key } from '~/generated/CommentsBottomSheetList.graphql';
 
 type CommentsListProps = {
+  activeCommentId?: string;
   onLoadMore: () => void;
   commentRefs: CommentsBottomSheetList$key;
 };
 
-export function CommentsBottomSheetList({ commentRefs, onLoadMore }: CommentsListProps) {
+export function CommentsBottomSheetList({
+  activeCommentId,
+  commentRefs,
+  onLoadMore,
+}: CommentsListProps) {
   const comments = useFragment(
     graphql`
       fragment CommentsBottomSheetList on Comment @relay(plural: true) {
+        dbid
         ...CommentsBottomSheetLineFragment
       }
     `,
     commentRefs
   );
 
-  const renderItem = useCallback<ListRenderItem<(typeof comments)[number]>>(({ item: comment }) => {
-    return (
-      <View className="mb-4">
-        <CommentsBottomSheetLine commentRef={comment} />
-      </View>
-    );
-  }, []);
+  const ref = useRef<FlashList<(typeof comments)[number]>>(null);
+
+  useEffect(() => {
+    if (activeCommentId && ref.current) {
+      const index = comments.findIndex((comment) => comment.dbid === activeCommentId);
+      if (index !== -1) {
+        setTimeout(() => {
+          if (!ref.current) {
+            return;
+          }
+          ref.current.scrollToIndex({ index, animated: true });
+        }, 200);
+      }
+    }
+  }, [activeCommentId, comments]);
```

Comment:
Add a delay to make sure the comments was fully loaded and we can scroll to specific comment if its out of view